packages/edit-post/src/components/keyboard-shortcut-help-modal/config.js

```diff
@@ -124,6 +124,10 @@ const blockShortcuts = {
 			/* translators: The forward-slash character. e.g. '/'. */
 			ariaLabel: __( 'Forward-slash' ),
 		},
+		{
+			keyCombination: access( 'm' ),
+			description: __( 'Switch between Visual Editor and Code Editor.' ),
```

Comment:
It'd be good to differentiate this from the other shortcut a bit more. How about something like the following:
```suggestion
			description: __( 'Switch between editing the block visually and editing its HTML.' ),
```

Reply:
@talldan is ctrl('m') a better option than access( 'm' )?